libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmBaseDomainField.tsx

```diff
@@ -101,6 +102,7 @@ export const OcmBaseDomainField = ({ managedDomains }: { managedDomains: Managed
             placeholder="example.com"
             isDisabled={useRedHatDnsService}
             isRequired
+            onChange={() => clearAlerts()}
```

Comment:
```suggestion
            onChange={clearAlerts}
```

Reply:
I think I'll change the code to clear the alerts  when validations are re-triggered